server/lib/s3backup/__init__.py

```diff
@@ -112,6 +119,22 @@ def put_object(self, Bucket=None, Key=None, Body=None, ContentMD5=None):
             with open('{}/{}/{}'.format(self.path, self.bucket_name, Key), 'wb') as f:
                 f.write(Body.read())
 
+    def upload_fileobj(self, read_content=None, Bucket=None, Key=None, Config=None, ExtraArgs=None):
```

Comment:
... and maybe these two as well. but we'll have to take a closer look at the unit tests.